### DISCUSSION TITLE
阿里云上传限速

### DISCUSSION THREAD

**Discussion by skrik2** [AlistGo/alist#6874] [General]
离线下载文件，会卡在 transferring ，进度条没反应

过很久一段时间，能在阿里云 app 中看到上传的文件

alist 中会有一个成功的字样，但是进度条无任何变化

上传时间与之前相比大大增加

开了会员和买那个第三方的包会不会好一点？

**Comment by anwen-anyi** [comment]
上传最终取决于你搭建AList设备的上传速度

**Reply by skrik2** [reply]
你有没有做测试？

不是直接上传那个选项，**是离线下载后上传的那个选项**


我在国内（广东）服务器，国外（新加坡）服务器，本地电脑（1000m），本地电脑阿里网盘客户端，四个环境做了测试。

确实变慢了。

- 这个月和之前上传速度对比，差距太大。

- 我 onedrive 有会员，两者上传速度一对比，alist阿里网盘惨不忍睹

- 阿里网盘客户端和alist上传速度对比，差距也很大

阿里网盘我没开会员，不知道开会员，alist 用起来会不会好一点

**Reply by skrik2** [reply]
![Snipaste_2024-07-26_13-22-14](https://github.com/user-attachments/assets/550376e4-06f9-4d80-a85a-b1d1ab51b89d)

刚刚我又测试了一下

还是老样子。
上传一段时间后，显示成功了，但是进度没拉满

上传慢，总比不能上传好，凑合着用吧
